services/validator.service.js

```diff
@@ -0,0 +1,73 @@
+const Joi = require('@hapi/joi');
+const { MD5 } = require('crypto-js');
+const ERROR = require('../dictionary/errors.dictionary');
+
+const MIN = 0;
+const MAX = 16;
+const MIN_NAME = 3;
+const MIN_PASS = 6;
+const MIN_RATE = 1;
+const MAX_RATE = 5;
+const MINOR = 18;
+const DATED_LENGTH = 10;
+const DATED_REGEX = /\d{2}\/\d{2}\/\d{4}/i;
+
+const CRUSH_SCHEME = Joi.object({
+  name: Joi.string().min(MIN_NAME).required().messages({
+    'string.min': ERROR.INVALID_NAME,
+    'any.required': ERROR.NO_NAME,
+  }),
+  age: Joi.number().min(MINOR).required().messages({
+    'number.min': ERROR.INVALID_AGE,
+    'any.required': ERROR.NO_AGE,
+  }),
+  date: Joi.object({ 
+    datedAt: Joi.string().pattern(DATED_REGEX).min(DATED_LENGTH)
```

Comment:
Sua rúbrica tá sendo usada, @LizzardMedeiros, nada mais justo hahaha
Seria tipo assim?
```suggestion
    datedAt: Joi.date()
  ```